src/controllers/commands/CommandController.cpp

```diff
@@ -434,6 +434,20 @@ void CommandController::initialize(Settings &, Paths &paths)
         userPopup->show();
         return "";
     });
+
+    this->registerCommand(
+        "/chatters", [](const auto & /*words*/, auto channel) {
+            auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+
+            auto message =
+                (twitchChannel == nullptr)
+                    ? "The /chatters command only works in Twitch Channels"
+                    : "Chatter count: " +
+                          QString::number(twitchChannel->chatterCount());
+
+            channel->addMessage(makeSystemMessage(message));
```

Comment:
use an if case instead of a ternary operator here - readability is more important than compactness